client/my-sites/woocommerce/dashboard/required-plugins-install-view.jsx

```diff
@@ -271,6 +323,9 @@ class RequiredPluginsInstallView extends Component {
 				slug: workingOn,
 			};
 
+			// Progress: install a single plugin.
+			this.resetTimeoutTimer( TIME_TO_PLUGIN_INSTALLATION );
+			// installPlugin actually activates it too.
```

Comment:
This comment is helpful. You could go one step further... get rid of the comment and instead `import { installPlugin as installAndActivatePlugin }` instead.

Reply:
Brilliant suggestion, thanks!